examples/pytorch/01-Getting-started.ipynb

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
<p>do we need this entire block again?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
I am not sure -- was just copying over what we have on the TF side, we follow the same pattern there